### DISCUSSION TITLE
Unexpected Deletions with no apparent reason

### DISCUSSION THREAD

**Discussion by Billbarasch** [phw198/OutlookGoogleCalendarSync#1916] [Q&A]
I had 2 events deleted by OGCS that appeared in both google calendar and in outlook calendar. 
I don't see any reason for the deletions. I noticed the the sync window the deletions so I recovered manually the apportments. 
What is the logic behind how OGCS decided to delete these events. 
Thanks for any help

**Comment by phw198** [comment]
Many and varied, but at its simplest level that the item doesn't exist in the "source" calendar and is therefore deleted in the "target" calendar. If you say the item existed in both calendars, I'd likely need to review your logfile to understand why the deletion occurred.

**Reply by Billbarasch** [reply]
Hi, I am doing bidirectional sync between google and outlook so both are source and destination to my understanding. I should have mentioned that in my post. Again any reason why this would occur.

**Reply by phw198** [reply]
> so both are source and destination to my understanding

No not true - it depends on when, where and how the calendar item was first created as to what happens for it to sync. If you copy and pasted an already synced Outlook item whilst OGCS was not running, then that would be a [known issue](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Limitations#outlook-appointment-copy--paste).

For further triage, you would need to [file a bug report](https://github.com/phw198/OutlookGoogleCalendarSync/issues/new/choose) and provide the requested information in order for proper diagnosis of the root cause (which includes working out how to replicate the issue so it can be seen in the logs).
